addon/components/hyper-table-v2/filtering-renderers/common/facets-loader.hbs

```diff
@@ -2,17 +2,11 @@
   {{#if this.searchEnabled}}
     <div class="margin-bottom-px-6">
       <label>{{t "hypertable.column.filtering.search_term.label"}}</label>
-      <OSS::InputContainer
+      <OSS::SearchField
         @value={{this.searchQuery}}
         @onChange={{this.onInputChanged}}
         @placeholder={{this.searchPlaceholder}}
-      >
-        <:suffix>
-          {{#if (gt this.searchQuery.length 0)}}
-            <OSS::Icon @icon="fa-times" role="button" {{on "click" this.onClearSearch}} />
-          {{/if}}
-        </:suffix>
-      </OSS::InputContainer>
+      />
```

Comment:
I think you can also add a data-control-name here too :) 
Maybe `data-control-name="filter-renderers-facets-loader-search-input"`